userprofile.py

```diff
@@ -41,11 +41,11 @@ def add_pokemon(self, pokemon: Pokemon):
 			return
 		self.pokemon.append(pokemon._id)
 
-	def get_embed(self) -> discord.Embed:
+	def get_embed(self, name) -> discord.Embed:
```

Comment:
Should not take any arguments, all information in the embed should be available via `self`.